src/github.com/getlantern/flashlight/app/mock_autoupdate.go

```diff
@@ -0,0 +1,15 @@
+// +build mockupdate
+
+package app
+
+// This is a public key derived from a private key that is freely available.
+// Don't use in production.
+const packagePublicKey = `-----BEGIN PUBLIC KEY-----
```

Comment:
That's probably the cause, this PR was merged into `devel`. Not sure is this is already on `pro-backend`.